components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/handlers/grant/AbstractAuthorizationGrantHandler.java

```diff
@@ -1103,4 +1120,20 @@ private boolean isFederatedUser(OAuthTokenReqMessageContext tokReqMsgCtx) {
         }
         return tokReqMsgCtx.getAuthorizedUser().isFederatedUser();
     }
+
+
+    private void setACRValueInTheToken(OAuthTokenReqMessageContext tokenReqMsgCtxt, AccessTokenDO accessTokenDO) {
+
+        Optional<String> optionalSelectedACR = OAuth2Util.getSelectedACRValue(tokenReqMsgCtxt);
+        if (!optionalSelectedACR.isPresent()) {
+
+            log.debug("ACR value is not available for the token issued for '"
```

Comment:
shall we add isDebugEnabled check

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1984/commits/59ba3e10b73d02dcf0d68de230df5add71a2ab97